script/Ciel/Day 5/05_01_CIEL05_3B.txt

```diff
@@ -64,7 +65,7 @@ With an amused look on her face, Noel-sensei leaps away while vigorously waving
 [sha:b188261c0936ea56c1300b95681f07b369e57a9d]{
 -- Page 441, Offset 12037.
 -- ……本当に、この傾斜になった瓦礫の山をびょんびょんと跳んでいった。
-She's seriously bounding right up the side of that steep mountain of debris...
+She's %{i}seriously%{/i} bounding right up the side of that steep mountain of debris...
```

Comment:
Not sure I love this italics, I read it as her working really hard / going at the act of bounding with full force, rather than "Damn, she's really bounding right up it like it's nothing". Maybe tweak 本当 a bit to
> I can't believe she's bounding straight up the side of that...

Reply:
suggestion works for me